libs/application/templates/new-primary-school/src/fields/DynamicDisabledText/index.tsx

```diff
@@ -0,0 +1,65 @@
+import {
+  FieldComponents,
+  FieldTypes,
+  FormTextWithLocale,
+} from '@island.is/application/types'
+import { TextFormField } from '@island.is/application/ui-fields'
+import { useLocale } from '@island.is/localization'
+import { FC, useEffect } from 'react'
+import { useFormContext } from 'react-hook-form'
+import { Application, Field } from '@island.is/application/types'
+import { formatTextWithLocale } from '@island.is/application/core'
+import { Locale } from '@island.is/shared/types'
+
+interface DynamicDisabledTextProps {
+  application: Application
+  field: Field & {
+    props: {
+      value: FormTextWithLocale
+    }
+  }
+}
+
+const DynamicDisabledText: FC<DynamicDisabledTextProps> = ({
```

Comment:
Doesn't it make more sense to remove `application` from the interface and instead put `FC<React.PropsWithChildren<FieldBaseProps & DynamicDisabledTextProps>>`? 🤔 

```suggestion
interface DynamicDisabledTextProps {
  field: {
    props: {
      value: FormTextWithLocale
    }
  }
}

const DynamicDisabledText: FC<React.PropsWithChildren<FieldBaseProps & DynamicDisabledTextProps>> = ({
```

Reply:
Fixed